src/ape/api/transactions.py

```diff
@@ -294,6 +305,18 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs(self) -> Iterable[Tuple[Any]]:
```

Comment:
I think we can make this more streamlined
```suggestion
    def debug_logs(self) -> Iterable[str]:
```

Reply:
They're not all strings

Reply:
(just noticed the below)

Wouldn't the native types be more useful rather than making them all flat strings?

Reply:
Have to convert them somewhere, rather do it before caching it than afterwards

Reply:
`show_debug_logs()` is pretty unlikely to be accessed multiple times.  The `debug_logs` prop might be if leveraged in a user script or console.  Could cache both if you think it would be beneficial.

As a programmer, getting access to typed data is significantly more useful to me than a list of strings.  Converting `debug_logs` would essentially prevent that access or force them to re-implement what we already have.  Maybe we could compromise and provide both?

```python
    @cached_property
    def debug_logs_typed(self) -> Iterable[Tuple[Any]]:
        ...

    @cached_property
    def debug_log_lines(self) -> Iterable[str]:
        ...
```